lib/ui/page/home/page/my_profile/delete_email/controller.dart

```diff
@@ -62,9 +60,14 @@ class DeleteEmailController extends GetxController {
         s.editable.value = false;
         s.status.value = RxStatus.loading();
         try {
-          await _myUserService.removeUserEmail(email, password: password);
-          pop?.call();
+          final code = ConfirmationCode.tryParse(s.text);
```

Comment:
What if my password contains 4 digits (requirements for `ConfirmationCode`)? Isn't that possible?